web/viewer.css

```diff
@@ -503,6 +543,16 @@ select {
   margin-bottom: -4px;
 }
 
+#editorInkParamsToolbar {
+  inset-inline-end: 195px;
+  background-color: var(--toolbar-bg-color);
+}
+
+#editorFreeTextParamsToolbar {
+  inset-inline-end: 223px;
+  background-color: var(--toolbar-bg-color);
+}
```

Comment:
Unfortunately hard-coding these positional values will *not* work in general, since I'd expect that they'll even be incorrect in e.g. the *built-in* Firefox PDF Viewer (since the `OpenFile` button isn't included there).

Furthermore, they the viewer is resized some of the toolbarButtons in the `toolbarViewerRight`-div will be hidden and again the position will become incorrect.

The, by far, easiest solution here would probably be to *move* (in the HTML code) all of the "new" editor-related buttons such that the toolbar looks something like the following mock-up:

![buttons](https://user-images.githubusercontent.com/2692120/173841828-9dec89c4-1527-477d-9b9f-b08cf2db7424.png)

Doing that would ensure that the editor-buttons don't around at all (and would avoid us having to add a bunch of JS code to reposition the new editor-toolbars).

---

*For future reference:* Before editing is enabled by default, we also need to update the existing CSS `@media`-rules that handle hiding of toolbar buttons when the viewer becomes narrow.
This is obviously not something that we need to worry about here, but I wanted to mention it nonetheless.